docs/sources/mimir/release-notes/v2.14.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Grafana Mimir version 2.14 release notes"
+menuTitle: "V2.14 release notes"
+description: "Release notes for Grafana Mimir version 2.14"
+weight: 1100
+---
+
+# Grafana Mimir version 2.14 release notes
+
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.14 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+The minimal supported version of Go is updated to 1.22.
```

Comment:
Is this relevant to users? They just use the binary we build, right?

Reply:
As Charles says, this is not relevant. Furthermore mimir binaries are built with Go 1.23.1.